hms-api/src/repository/domain/Hackathon.ts

```diff
@@ -79,15 +83,11 @@ class Hackathon {
       result.addFailure('description is null');
     }
 
-    if (!this.startDate) {
-      result.addFailure('startDate is null');
-    }
-
-    if (!this.endDate) {
-      result.addFailure('endDate is null');
-    }
-
-    if (this.startDate >= this.endDate) {
+    if (
```

Comment:
this is shorter, yes, but you lose a bit of information this way

Reply:
I'm guessing you did this because the `if (this.startDate >= this.endDate)` might not work right when either `this.startDate` or `this.endDate` is null.  In that case you might want to change just that one if statement

Reply:
ah now I see, because startDate and endDate can be null... hmmm... I think that isn't a good change, because now it is possible to make a Hackathon with no start or end date